src/bouwdossiers/batch.py

```diff
@@ -89,16 +89,14 @@ def _normalize_bestand(bestand):
 
 
 def get_access(el):
-    beperking = el.get('openbaarheidsBeperking')
-    if beperking:
-        if beperking == 'N':
-            return models.ACCESS_PUBLIC
-        return models.ACCESS_RESTRICTED
-    else:
-        openbaar = el.get('openbaar')
-        if openbaar == 'J':
-            return models.ACCESS_PUBLIC
+    if (
+        el.get('openbaarheidsBeperking', 'N').lower() == 'j'
```

Comment:
Als deze parameters niet gedefinieerd zijn (of er staat een spelfout in de namen in deze 4 regels) zijn de defaults ingesteld om de access altijd public te maken. Dat lijkt me niet handig. Kun je de defaults niet beter andersom zetten om te voorkomen dat je persongeluk te veel public maakt?